README.md

```diff
@@ -34,7 +34,8 @@ Please ensure you are using the correct version of MixinGradle for your ForgeGra
 | ForgeGradle Version | Mixin Version   | MixinGradle Version To Use |
 | ------------------- | --------------- | -------------------------- |
 | 2.3                 | 0.8 and below   | `0.6-SNAPSHOT`             |
-| 3.0                 | 0.8             | `0.7-SNAPSHOT`             |
+| 3.0 - 4.1           | 0.8             | `0.7-SNAPSHOT`             |
+| 5.0 and above       | 0.8.2           | `0.8-SNAPSHOT`             |
```

Comment:
Probably should be 0.7-SNAPSHOT.
But yes looks good on everything else to me.
No breaking changes, the annotations exist for all older gradles. Looks good.